octobot_trading/exchanges/exchange_manager.py

```diff
@@ -81,29 +82,36 @@ def __init__(self, config, exchange_class_string):
     async def initialize_impl(self):
         await exchanges.create_exchanges(self)
 
-    async def stop(self, warning_on_missing_elements=True):
+    async def stop(self, warning_on_missing_elements=True, enable_logs=True):
```

Comment:
I don't really like this approach, why not using debug level?

Reply:
We are already on debug. I added that because we are creating temporary exchange managers when checking user accounts (to display credentials validity and exchange support status). We are creating an exchange manager to use the unified exchange builder system (and instanciate the exchange with the right tentacle, config etc). Because of that we are creating as many exchange managers as there are exchanges in the person's config and logging exchange stop creates a ton of useless logs

Reply:
I see. Thanks for the explanation.